about/index.md

```diff
@@ -5,16 +5,16 @@ weight: 10
 
 <img src="http://i.imgur.com/cADD79P.png" alt="Buoy: Tell your friends where you are and what you need." align="right" width="300" />
 
-Buoy is a free and open source, decentralized tool designed to quickly connect users with trusted friends, family members, advocates, and other allies in times of need.
+Buoy is a crisis response platform that connects families, friends, and teams during emergency situations. 
 
-Buoy is a community-based crisis response system for any emergency in which someone needs help but might not want to -- or can't -- call 911.
+Buoy is a free tool designed to quickly to quickly alert and communicate with a prescribed group of people when assistance is needed.
 
-Buoy allows users to pre-define a custom network of trusted friends and allies. In an emergency, the user's Response Team can be alerted with one touch. Each crisis alert provides Responders with a real-time map of the Alerter's and each others' locations, a chat room in which they can all coordinate a response, and media (photo, video, audio) sharing capabilities.
+Buoy allows users to to create customized networks of trusted friends and allies and to plan responses for emergency situations. In the case of an emergency, the user's Response Team can be alerted with one touch. The alert creates a real-time map of each team member's location, a live chat to coordinate a response, and live photo, audio, and video streaming capabilities.
```

Comment:
Technically, there's only video streaming. Photo sharing exists but isn't streamed. This is a technical distinction that matters to some people (me) but most people won't really understand the nuance. Is there a way to more clearly describe this without being technically inaccurate?